helix-loader/src/grammar.rs

```diff
@@ -167,7 +163,10 @@ fn fetch_grammar(grammar: GrammarConfiguration) -> Result<()> {
             // Fetch the exact revision from the remote.
             // Supported by server-side git since v2.5.0 (July 2015),
             // enabled by default on major git hosts.
-            git(&grammar_dir, ["fetch", REMOTE_NAME, &revision])?;
+            git(
+                &grammar_dir,
+                ["fetch", "--depth", "1", REMOTE_NAME, &revision],
```

Comment:
I was wondering why it was taking so long to fetch compared to the submodules or script, can't believe I forgot about this :smile: